server/secrets.py

```diff
@@ -0,0 +1,3 @@
+# MBTA v3 API key here; optionally can be injected via environment variable
```

Comment:
Useful trick that might be worth mentioning here.
`git update-index --assume-unchanged server/secrets.py` will let you change this file and then hide it from `git status` and `git add --all` and others.
I have it aliased in my `~/.gitconfig` as
```
[alias]
  hide = update-index --assume-unchanged
  unhide = update-index --no-assume-unchanged
  hidden = !git ls-files -v | grep "^[[:lower:]]"
```